cooperate.sql

```diff
@@ -0,0 +1,61 @@
+DROP SCHEMA public CASCADE;
+CREATE SCHEMA public;
+
+DROP DATABASE cooperate;
+CREATE DATABASE cooperate;
+
+\c cooperate;
+--This table contains login info for user as well as their karma
+
+CREATE TABLE Users (
+  id varchar PRIMARY KEY NOT NULL, --username
```

Comment:
There doesn't need to be a separate id for each table that's referenced in the review table. We can, for example, make the usernames the primary key for the users and then use that as a foreign key in the review table to save complexity.

Reply:
I agree, using the foreign keys made the code much more readable and simpler. I added in the changes, please let me know if there are any other issues.

Reply:
I agree that it isn't really elegant to have to create a column in one line and then declare it as a foreign key in a separate line. Unfortunately, it looks like there isn't any good way to do it in one line in postgres which is pretty strange: https://dba.stackexchange.com/questions/132029/how-to-add-a-column-with-a-foreign-key-constraint-to-a-table-that-already-exists